Header header
float64 global_time
float64 filtering_time
float64 surface_normal_estimation_time
float64 keypoint_selection_time
float64 initial_pose_estimation_time
float64 pointcloud_registration_time
float64 correspondence_estimation_time_for_all_matchers
float64 transformation_estimation_time_for_all_matchers
float64 transform_cloud_time_for_all_matchers
float64 cloud_align_time_for_all_matchers
float64 outlier_detection_time
float64 registered_points_angular_distribution_analysis_time
float64 transformation_validators_time
float64 covariance_estimator_time
float64 map_update_time
